forge-gui/res/cardsfolder/n/nameless_race.txt

```diff
@@ -13,6 +13,6 @@ SVar:Limit:SVar$Active/Plus.Buried
 SVar:Active:Count$Valid Permanent.White+nonToken+OppCtrl
 SVar:Buried:Count$ValidGraveyard Card.White+OppCtrl
 SVar:LifePaidOnETB:Number$0
-AI:RemoveDeck:All
+SVar:NeedstoPlayVar:Limit GE1
```

Comment:
did you test this?
Imo probably still too difficult for AI because own life total is an important factor too